tpke/src/ciphertext.rs

```diff
@@ -40,7 +40,6 @@ impl<E: PairingEngine> Ciphertext<E> {
         let mut bytes = Vec::new();
         self.commitment.write(&mut bytes).unwrap();
         self.auth_tag.write(&mut bytes).unwrap();
-        bytes.extend_from_slice(&self.ciphertext);
```

Comment:
oops! it seems you removed here the ciphertext from the serialized output!

Reply:
I find it very worrisome that this wasn't caught by tests.